debian/patches/0001-bitlbee.conf.patch

```diff
@@ -1,5 +1,12 @@
---- bitlbee.conf	2009-06-01 00:20:24.000000000 +0100
-+++ bitlbee.conf	2009-06-07 21:16:19.000000000 +0100
+
```

Comment:
Why this change? The current patch file could use improvements e.g. by adding DEP-3 tags, but changing from one arbitrary patch format (AFAICT) to another seems unnecessary.